sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceAsyncClient.java

```diff
@@ -48,26 +48,173 @@
 import static com.azure.data.tables.implementation.TableUtils.swallowExceptionForStatusCode;
 
 /**
+ *
  * Provides an asynchronous service client for accessing the Azure Tables service.
  *
+ * <h2>Overview</h2>
+ *
  * <p>The client encapsulates the URL for the Tables service endpoint and the credentials for accessing the storage or
  * CosmosDB table API account. It provides methods to create, delete, and list tables within the account. These methods
  * invoke REST API operations to make the requests and obtain the results that are returned.</p>
  *
- * <p>Instances of this client are obtained by calling the {@link TableServiceClientBuilder#buildAsyncClient()} method
- * on a {@link TableServiceClientBuilder} object.</p>
+ * <h2>Getting Started</h2>
+ *
+ * <p>The building and authenticating of instances of this client are handled by {@link TableServiceClientBuilder} instances. The sample below
+ * shows how to authenticate and build a TableServiceAsyncClient using a connection string.</p>
  *
- * <p><strong>Samples to construct an async client</strong></p>
- * <!-- src_embed com.azure.data.tables.tableServiceAsyncClient.instantiation -->
+ * <!-- src_embed com.azure.data.tables.tableServiceAsyncClient.instantiation.connectionstring -->
  * <pre>
  * TableServiceAsyncClient tableServiceAsyncClient = new TableServiceClientBuilder&#40;&#41;
- *     .endpoint&#40;&quot;https:&#47;&#47;myvault.azure.net&#47;&quot;&#41;
- *     .credential&#40;new AzureNamedKeyCredential&#40;&quot;name&quot;, &quot;key&quot;&#41;&#41;
+ *     .connectionString&#40;&quot;connectionstring&quot;&#41;
  *     .buildAsyncClient&#40;&#41;;
  * </pre>
  * <!-- end com.azure.data.tables.tableServiceAsyncClient.instantiation -->
  *
+ * <p>See {@link TableServiceClientBuilder} documentation for more information on constructing and authenticating a client.</p>
+ *
+ * <p>The following code samples show the various ways you can interact with the tables service using this client.</p>
+ *
+ * <hr/>
+ *
+ * <h3>Create a Table</h3>
+ *
+ * <p>The {@link #createTable(String) createTable} method can be used to create a new table within an Azure Storage or Azure Cosmos account.
+ * It returns a TableClient for the newly created table.</p>
+ *
+ * <p>The following sample creates a table with the name "myTable".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.tableServiceAsyncClient.createTable#String -->
+ * <pre>
+ * tableServiceAsyncClient.createTable&#40;&quot;myTable&quot;&#41;
+ *     .contextWrite&#40;Context.of&#40;&quot;key1&quot;, &quot;value1&quot;, &quot;key2&quot;, &quot;value2&quot;&#41;&#41;
+ *     .subscribe&#40;tableAsyncClient -&gt;
+ *         System.out.printf&#40;&quot;Table with name '%s' was created.&quot;, tableAsyncClient.getTableName&#40;&#41;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.tableServiceAsyncClient.createTable#String -->
+ *
```

Comment:
Add here and for other samples,
 * <p><strong>Note:</strong> for synchronous sample, refer to {@link sync client}</p>